test/cucumber/Notificacao.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Notification
+    As an administrator
+    I want to be notified about the students issues without searching for their historic
+    So I can know who has its performance under the class average quickly
+
```

Comment:
Correctly follows the Feature template which introduces the User Story. Just put "class average grades", because the focus is on the students average grades